EventHubs/target/metrics_build/EventHubs_Metrics/sumoclient.js

```diff
@@ -125,10 +125,11 @@ SumoClient.prototype.flushBucketToSumo = function(metaKey) {
     var self = this;
     let curOptions = Object.assign({},this.options);
 
-    this.context.log("Flush buffer for metaKey:"+metaKey);
+    // this.context.log("Flush buffer for metaKey:"+metaKey);
```

Comment:
so we have files present in https://github.com/SumoLogic/sumologic-azure-function/blob/master/sumo-function-utils/lib/sumoclient.js 
Files in target folder are automatically generated by the build. so change the files in sumo-function-utils folder as well.

Reply:
Perform the steps and update the files